fontique/src/backend/coretext.rs

```diff
@@ -72,23 +68,23 @@ impl SystemFonts {
         let key = key.into();
         let sample = key.script().sample()?;
         let font = create_fallback_font_for_text(sample, key.locale(), false)?;
-        let family_name = unsafe { CTFontCopyFamilyName(&font) };
+        let family_name = unsafe { font.family_name() };
```

Comment:
Is this unsafe only because the safety invariants haven't been checked on the objc2 side?

Reply:
They have to be analyzed on the objc2 side by a human and marked as being able to be safe.